data/json/monsters/fish.json

```
@@ -1354,11 +1354,11 @@
   {
     "id": "mon_fish_porgy",
     "type": "MONSTER",
-    "copy-from": "mon_fish_small",
+    "copy-from": "mon_fish_tiny",
     "name": { "str_sp": "scup" },
     "description": "Also known as porgy, the scup is a small fish commonly found in Atlantic coastal waters, recognizable by its silver body with blue and purple tinges.",
-    "volume": "1300 ml",
-    "weight": "550 g",
+    "volume": "770 ml",
+    "weight": "340 g",
```

```suggestion
    "weight": "340 g",
```
Scup are lean and tall fish, with a small mouth, high set eyes, and one long spiny dorsal fin. They can live up to at least 14 years, weigh up to 5 pounds, and grow up to 18 inches, although most of the catch in Massachusetts consists of fish less than 6 years old, 3 pounds in weight, and 14 inches in length.  https://www.mass.gov/info-details/learn-about-scup#:~:text=Scup%20are%20lean%20and%20tall,and%2014%20inches%20in%20length.

---

On the link you have in the summary notes I went to that website and looked up scup. It lists scup average weight at 2.1kg
 
https://fishbase.se/summary/SpeciesSummary.php?ID=452&AT=scup

---

5 lbs in weight is WAY above what I find from marine biology resources I trust. The max weight according to fishbase is 4.63 lbs and that's *the record*. So I have a very hard time trusting this. The size here is calculated from a roughly 10" individual, though my weight calculation is imperfect and it's entirely possible this one may be too light.